internal/common/baseclient/iigo.go

```diff
@@ -29,36 +31,47 @@ func (c *BaseClient) RuleProposal() string {
 }
 
 // GetClientPresidentPointer is called by IIGO to get the client's implementation of the President Role
+// COMPULSORY: ovverride to return a pointer to your own President object
 func (c *BaseClient) GetClientPresidentPointer() roles.President {
 	return &BasePresident{}
 }
 
 // GetClientJudgePointer is called by IIGO to get the client's implementation of the Judge Role
+// COMPULSORY: ovverride to return a pointer to your own Judge object
 func (c *BaseClient) GetClientJudgePointer() roles.Judge {
 	return &BaseJudge{}
 }
 
 // GetClientSpeakerPointer is called by IIGO to get the client's implementation of the Speaker Role
+// COMPULSORY: ovverride to return a pointer to your own Speaker object
 func (c *BaseClient) GetClientSpeakerPointer() roles.Speaker {
 	return &BaseSpeaker{}
 }
 
+// BASE
```

Comment:
what does this func actually do? document it? I think @mpardalos wrote this actually...

Reply:
I was just wondering this... I'm inclined to remove it as there is no case that tax wouldn't be taken exactly as expected

Reply:
i've made an issue

Reply:
#261